src/triage/experiments/multicore.py

```diff
@@ -65,6 +65,8 @@ def catwalk(self):
                 self.n_processes
             ):
                 model_ids += batch_model_ids
+            # remove Nones from model_ids so we don't try to predict/evaluate on skipped baselines
+            model_ids = list(filter(None.__ne__, model_ids))
```

Comment:
Strong preference to avoid unnecessary use of the "magic" or "double-underscore" methods.

If you just want to drop "false-y" items, `list(filter(None, model_ids))` will do the trick.

Generally, though, if you don't have a predicate in hand, a comprehension does well:

    [model_id for model_id in model_ids if model_id is not None]

Reply:
yeah, not a fan of this, either. `list(filter(None, model_ids))` will also remove `0`, but i think either a lambda or list comprehension for `!= None` should work as an alternative.